tsconfig.prod.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
```

Comment:
Shall sourceMap be set to false here?

Reply:
no need, it is false by default

Reply:
but happy to add it to make it more clear